scripts/workers/code_upload_submission_worker.py

```diff
@@ -262,7 +272,7 @@ def create_static_code_upload_submission_job_object(message):
         value=str(submission_meta["submission_time_limit"]),
     )
     SUBMISSION_TIME_DELTA_ENV = client.V1EnvVar(
-        name="SUBMISSION_TIME_DELTA", value="3600"
+        name="SUBMISSION_TIME_DELTA", value="1"
```

Comment:
This value to should be 1 hour. Let's not check in values like 1 second for testing purposes only

Reply:
@Ram81 actually this is set on purpose.
Assume the following situation.

- time_delta is 1 hr
- submission worker stops after 15 mins and saving the file.
- whenever code_upload_worker would receive the message and see submission worker is completed but sidecar is running. So it would notify EvalAI that no submission file was found.
- So in order to support delete of job and updation to evalai, we need to have it 1 sec.
- Because the message is checked every 0.5 secs in code upload worker now.